KarhooUISDK/API/KarhooComponents.swift

```diff
@@ -21,8 +21,8 @@ public class KarhooComponents: BookingScreenComponents {
             cornerRadious: UIConstants.CornerRadius.large,
             borderLine: true,
             dropShadow: false,
-            verticalPadding: 5.0,
-            horizontalPadding: 5.0,
+            verticalPadding: 0,
+            horizontalPadding: 0,
```

Comment:
Why the change in padding?

Reply:
5.0 margin resulted in misalignment in QuoteList. There is no reason to use internal padding - if design require so, we or DP can use external layout to handle that.